docs/how_to/preserve_hosts.md

```diff
@@ -0,0 +1,81 @@
+# How to Preserve Hosts
+
+## Motivation
+
+Often when developing acceptance tests to be run with beaker, you'll want to be
+able to quickly iterate on those tests in an already setup System Under Test
+(SUT) configuration. Beaker provides the ability to do that using its preserved
+hosts functionality.
+
+## How Do I Use It?
+
+Note that where you decide to use this option will affect its precedence. You
+can learn more about this in our
+[argument processing & precedence doc](../topics/argument_processing_and_precedence.md).
+
+### Command-Line Option
+
+The primary way that this option is provided is through the command line, where
+it's represented by the `--preserved-hosts` options. This option defaults to
+`never`, and if you run `beaker --help`, you'll see the range of values that can
+be provided:
+
+    $ beaker --help
+        ...
+        --preserve-hosts [MODE]      How should SUTs be treated post test
+                                     Possible values:
+                                     always (keep SUTs alive)
+                                     onfail (keep SUTs alive if failures occur during testing)
+                                     onpass (keep SUTs alive if no failures occur during testing)
+                                     never (cleanup SUTs - shutdown and destroy any changes made during testing)
+                                     (default: never)
+        ...
+
+### Local Options or Global Config
+
+You can also provide a value for this option wherever you can provide a value
+that gets merged into beaker's global options hash. The two ways to do this are
+to provide it in a local options file, or in the global `CONFIG` section of your
+hosts file. To learn more about how to set these options, check out the
+[Options File Values](../topics/argument_processing_and_precedence.md#options-file-values)
+and the
+[`CONFIG` Section of a Hosts File](../topics/argument_processing_and_precedence.md#config-section-of-hosts-file)
+sections of the
+[argument processing & precedence doc](../topics/argument_processing_and_precedence.md).
+
+Either way, the underlying property name that you will have to set if you use
+these options is `:preserve_hosts`.
+
+## What Does It Do (In Detail)?
+
+Using the preserved hosts functionality outputs a `hosts_preserved.yml` file into
+the directory that holds all of the log files for a particular beaker run. The
+short way to access that directory is to look into the `log/latest` path, which
+is a symlink to the actual log directory for the run.
+
+The differences between the run that generates a preserved hosts file and later
+runs are listed below:
```

Comment:
I'm a little bit confused here about the concept of a run here, and think you just mean the file instead of run. Maybe rephrase to say something more like "A preserved_hosts.yml changes these options from the initial run". I say this because a subsequent "run" could still be modified by any additional flags or ENV VARs set.